### PR TITLE
add new query to get submission state

### PR DESCRIPTION
**Pull request by jaserud**
**Description of changes**

connected prs: 
https://github.com/icgc-argo/argo-clinical/pull/344
https://github.com/icgc-argo/platform-ui/pull/994

**Type of Change**

- [ ] Bug
- [ ] Styling
- [x] New Feature

### CODE REVIEW

File: schemas/Clinical/index.js [hunk 1]
```diff
@@ -178,6 +178,11 @@ const typeDefs = gql`
     Retrieve current stored Clinical Submission Data Dictionary Schema version
     """
     clinicalSubmissionSchemaVersion: String!
+
+    """
+    Retrieve current Clinical Submission System disabled state
```
Comment (hlminh2000) [inline schemas/Clinical/index.js]: let's have add to the description that it's both submission and registration :)

File: schemas/Clinical/index.js [hunk 2]
```diff
@@ -178,6 +178,11 @@ const typeDefs = gql`
...
+    """
+    clinicalSubmissionSystemState: Boolean!
```
Comment (hlminh2000) [inline schemas/Clinical/index.js]: rename to something like `clinicalSubmissionSystemDisabled` or `clinicalSubmissionSystemEnabled`

File: services/clinical/index.js [hunk 3]
```diff
@@ -87,6 +90,19 @@ const getClinicalSubmissionSchemaVersion = async () => {
   return response.version;
 };
 
+/**
+ * @returns {Promise<boolean>}
+ */
```
Comment (hlminh2000) [inline services/clinical/index.js]: image

Comment (jaserud) [reply services/clinical/index.js]: looks like github does have the emoji so think of this as the party parrot... :taco:

**Pull request merged**